src/styles/globals.css

```diff
@@ -99,8 +99,7 @@ textarea {
   --contentPadding: var(--spacings-xLarge);
 }
 
-a,
-a:visited {
```

Comment:
Why was this removed?

Reply:
The reason for this was that this overrides the interactive color on the refresh button. From what I can see, this is necessary for the links in the footer, but that is already covered here: https://github.com/AtB-AS/planner-web/blob/main/src/layouts/shared/footer.module.css#L46. 

Let me know if you see any reason why we need this 👍

Reply:
I guess it's wanted in cases we have links in other places than e.g. the footer where it's covered. But not too big of a deal.